tests/unit_tests/admin/test_admin.py

```
@@ -0,0 +1,36 @@
+    with admin_blueprint.test_request_context('/admin/'):
+        session['user'] = None
+        response = admin_blueprint.view_functions['admin_panel']()
+        mock_render_template.assert_called_with('login.html', login_methods=Mock())
+
+@patch('platzky.admin.admin.render_template')
+def admin_panel_renders_admin_when_user_exists(mock_render_template, admin_blueprint):
+    mock_db = Mock()
+    mock_db.get_plugins_data.return_value = [{'name': 'plugin1'}, {'name': 'plugin2'}]
+    with admin_blueprint.test_request_context('/admin/'):
+        session['user'] = 'test_user'
+        response = admin_blueprint.view_functions['admin_panel']()
```

Ruff (0.8.2)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)